simpleclient_httpserver/src/main/java/io/prometheus/client/exporter/HTTPServer.java

```diff
@@ -42,9 +42,15 @@ protected ByteArrayOutputStream initialValue()
         }
     }
 
+    private static final String HEALTH_PATH = "/-/healthy";
```

Comment:
You can hardcode this.

Reply:
I don't actually like writing hardcode magic values. So this should be fine.

Reply:
I don't like having to indirect to find simple non-magic constants.